.internal-ci/helm/fog-services/templates/fog-ledger-rollout-cr.yaml

```diff
@@ -182,6 +184,10 @@ spec:
             affinity:
               {{- toYaml .Values.fogLedgerRouter.affinity | nindent 14 }}
             volumes:
+            {{- if eq .Values.fogLedger.persistence.enabled false }}
+            - name: fog-data
+              emptyDir: {}
+            {{- end }}
```

Comment:
:question: I'm understanding this to be an emptyDir when persistence isn't enabled, and per the docs, https://kubernetes.io/docs/concepts/storage/volumes/#emptydir it will go away on pod deletion.

When persistence is enabled then the VolumeClaimTemplate is used, https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#volume-claim-templates

Reply:
Thats correct.  There is a slight variation that with an `emptyDir {}`, if a container restarts on the same machine, the data from that `emptyDir` will remain.  If the data was just written inside the container (without and `emptyDir` volume), a restart would blow away that data.

Reply:
correct.  emptyDir will create the temp volume so the app can write to the directory but the data will not survive a pod delete or host move.